tests/foreman/cli/test_errata.py

```diff
@@ -714,757 +700,768 @@ def test_host_errata_search_commands(self):
-                )
-                # build a list of errata-id field value in the same order as
-                # received from the detailed sorted errata info list
-                sorted_errata_ids = [errata['errata-id'] for errata in sorted_errata_info_list]
-                # ensure that the errata ids received by Erratum.list is sorted
-                # as needed
-                self.assertEqual(errata_ids, sorted_errata_ids)
-
-    @pytest.mark.tier3
-    def test_positive_list_filter_by_product_id(self):
-        """Filter errata by product id
-
-        :id: 7d06950a-c058-48b3-a384-c3565cbd643f
-
-        :Setup: Errata synced on satellite server.
+ORG3_YUM_BIG_ERRATUM_COUNT = FAKE_3_YUM_ERRATUM_COUNT
+ORG3_BIG_ERRATA_ID = FAKE_3_ERRATA_ID
 
-        :Steps: erratum list --product-id=<productid>
 
-        :expectedresults: Errata is filtered by product id.
-        """
-        org_product_erratum_list = Erratum.list(
-            {'product-id': self.org_product['id'], 'per-page': 1000}
-        )
-        org_product_errata_ids = {errata['errata-id'] for errata in org_product_erratum_list}
-        errata_org_product_erratum_list = Erratum.list(
-            {'product-id': self.errata_org_product['id'], 'per-page': 1000}
-        )
-        errata_org_product_errata_ids = {
-            errata['errata-id'] for errata in errata_org_product_erratum_list
+@pytest.fixture(scope='class')
+def org1():
+    """an org with one custom product & repository"""
+    org1 = entities.Organization().create()
+    org1.product = entities.Product(organization=org1).create()
+    repo = make_repository(
+        {
+            'download-policy': 'immediate',
+            'organization-id': org1.id,
+            'product-id': org1.product.id,
+            'url': FAKE_6_YUM_REPO,
         }
-        self.assertEqual(len(org_product_errata_ids), self.org_product_erratum_count)
-        self.assertEqual(len(errata_org_product_errata_ids), self.errata_org_product_erratum_count)
-        self.assertIn(self.org_product_errata_id, org_product_errata_ids)
-        self.assertIn(self.errata_org_product_errata_id, errata_org_product_errata_ids)
-        self.assertEqual(org_product_errata_ids.intersection(errata_org_product_errata_ids), set())
-
-    @pytest.mark.tier3
-    def test_positive_list_filter_by_product_id_and_org_id(self):
-        """Filter errata by product id and Org id
-
-        :id: caf14671-d8b2-4a23-8c7e-6667bb78d4b7
-
-        :Setup: Errata synced on satellite server.
-
-        :Steps: erratum list --product-id=<product_id>
-            --organization-id=<org_id>
-
-        :expectedresults: Errata is filtered by product id and Org id.
-        """
-        product_erratum_list = Erratum.list(
+    )
+    Repository.synchronize({'id': repo['id']})
+    return org1
+
+
+@pytest.fixture(scope='class')
+def org2():
+    """an org with one custom product & repository"""
```

Comment:
Please use the docblocks to highlight the differences between `org1`, `org2`, etc fixtures

Reply:
I have improved docblocks a bit, please review once more